pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -270,6 +271,14 @@ func (k *Kubernetes) CreatePVC(name string, mode string) (*api.PersistentVolumeC
 	} else {
 		pvc.Spec.AccessModes = []api.PersistentVolumeAccessMode{api.ReadWriteOnce}
 	}
+	if strings.Contains(pvc.Name, "-") {
+		pvc.Name = strings.Replace(pvc.Name, "_", "-", -1)
+	}
+	for _, label := range pvc.Labels {
+		if strings.Contains(label, "_") {
+			pvc.Labels = transformer.ConfigLabels(strings.Replace(label, "_", "-", -1))
```

Comment:
A better idea here might be not to hardcode underscore-to-dash transition but instead to allow for an array of tuples (blacklisted char, replacement char) which will enable easier fix for this problem with other characters (if they show up) in the future.